encode.go

```diff
@@ -287,6 +287,11 @@ func (en *encoder) slice(key uint64, slval reflect.Value) {
 			packed.uvarint(v)
 		}
 
+	case []int:
+		for i := 0; i < sllen; i++ {
+			packed.svarint(int64(slt[i]))
+		}
+
 	case []int32:
 		for i := 0; i < sllen; i++ {
 			packed.svarint(int64(slt[i]))
```

Comment:
[https://github.com/golang/protobuf/blob/master/proto/encode.go#L732](url)
I am not so sure about the meaning of conversion to uint64 here, you guys may be more familiar with it :)